### DISCUSSION TITLE
Where do i find the eslint rules being applied

### DISCUSSION THREAD

**Discussion by parky128** [nrwl/nx#8042] [General]
Hello,

Where can I find the default ruleset being applied by Nx for eslint?

Thanks

**Comment by Trollyal** [comment]
Hey, have been asking the same question, to be honest, this should be a part of the docs ideally, and finding it was a treasure hunt of sorts, unless the placement is some convention, in which case i was th euneducated one.

Anyways, found it [in the repo here](https://github.com/nrwl/nx/tree/master/packages/eslint-plugin/src/configs)
I'll put it here for future treasure hunters
